models/response_model.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+type PageDescription struct {
```

Comment:
You should add comments (go doc) on both structs and fields so swagger could prepare documentation of it.

Reply:
this file was removed, but I will add the comments on the `response/response_model.go`

Reply:
added comments